project/Dependencies.scala

```diff
@@ -37,7 +37,9 @@ object Dependencies {
   val cats: ModuleID =           "org.typelevel"              %% "cats"            % "0.9.0"
   val enumeratum: ModuleID =     "com.beachape"               %% "enumeratum"      % "1.5.13"
 
-  val javaxServlet = "javax.servlet" % "servlet-api" % "2.5" % "provided"
+  //val javaxServlet = "javax.servlet" % "servlet-api" % "2.5" % "provided"
+  val javaxServlet2 = "javax.servlet" % "javax.servlet-api" % "4.0.1" % "provided"
```

Comment:
I mentioned this in person but I think we should limit ourselves to servlet 3.1. GAE says they support 2.5 and 3.1: https://cloud.google.com/appengine/docs/standard/java/runtime-java8#enhancements_and_upgrades_in_the_java_8_runtime